descriptions/SDK.Granny_3D.md

```diff
@@ -0,0 +1 @@
+[**Granny 3D**](http://www.radgametools.com/granny.html) has always made life easy for the programmers on your team. Whether you work in Max, Maya, or XSI, Granny makes it a snap to export, process, and animate anything that the artists can build. She has a data pipeline that's been battle-tested on every platform.
```

Comment:
it seems that their website has a web page that's more oriented to game engine, let's use that instead
```suggestion
[**Granny 3D**](http://www.radgametools.com/granny/sdk.html) has always made life easy for the programmers on your team. Whether you work in Max, Maya, or XSI, Granny makes it a snap to export, process, and animate anything that the artists can build. She has a data pipeline that's been battle-tested on every platform.
```